webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/DataTypes.scala

```diff
@@ -53,67 +53,71 @@ import zio._
 import zio.syntax._
 
 /**
- * A file that contains all the simple data types, like Version,
- * MemorySize, Manufacturer, etc.
+ * A file that contains all the simple data types, like Version, MemorySize, Manufacturer, etc.
  */
 
 /**
- * A simple class to denote a manufacturer
- * TODO : Should be merge with SoftwareEditor
+ * A simple class to denote a manufacturer TODO : Should be merge with SoftwareEditor
  */
-final case class Manufacturer(name:String) extends AnyVal
+final case class Manufacturer(name: String) extends AnyVal
 
 /**
  * A simple class to denote a software editor
  */
-final case class SoftwareEditor(val name:String) extends AnyVal
+final case class SoftwareEditor(val name: String) extends AnyVal
 
 sealed trait SecurityToken {
-  def key : String
+  def key: String
 }
 
 case object SecurityToken {
-  def kind(token : SecurityToken) = {
+  def kind(token: SecurityToken) =
```

Comment:
i really don't like this change - with explicit { } it's easier to see where something starts and ends, and avoid errors where we thing the code is for the method, but actually outside (or the other way round)

Reply:
it's now only for oneliners, which tends to make the code clearer (less bloated) in that case. For two lines or more, curly braces are kept.